app/views/docs/installation.phtml

```diff
@@ -61,14 +61,31 @@
     <pre class="line-numbers"><code class="prism language-bash" data-prism>docker-compose up -d --remove-orphans</code></pre>
 </div>
 
-<hr class="margin-bottom" />
```

Comment:
We removed HR, but never put it back. Should we put it back after uninstalling section?

Reply:
I think we can skip it